pkg/kubelet/server.go

```diff
@@ -296,6 +299,28 @@ func (s *Server) handlePods(w http.ResponseWriter, req *http.Request) {
 	w.Write(data)
 }
 
+// handleRunningPods returns a list of pods running on Kubelet. The list is
+// provided by the container runtime, and is different from the list returned
+// by handlePods, which is a set of desired pods to run.
+func (s *Server) handleRunningPods(w http.ResponseWriter, req *http.Request) {
```

Comment:
This is mostly a cut 'n paste of the above function.  Please factor out the common code.